src/Unit-2/lesson7/README.md

```diff
@@ -0,0 +1,129 @@
+# Unit 2, Lesson 7 - Statistics and some words about stale indexes! 
+
+At this point, you know a lot about storing, loading and querying documents using
+RavenDB, and that is awesome. Right?
+
+In this lesson you will learn how to get details of a just executed query.  
+
+## Introducing `RavenQueryStatics`
+
+Every time you perform a query using the client API, you can request
+some additional info using the `Statistics` method.
+
+````csharp
+RavenQueryStatics stats;
+
+var orders = (
+    from order in session.Query<Order>().Statistics(out stats)
+    where order.Company == "companies/1"
+    orderby order.OrderedAt
+    select order
+    ).ToList();
+
+Console.WriteLine($"Index used was: {stats.IndexName}");
+````
+
+The results are avaiable via a [`RavenQueryStatics` object](https://ravendb.net/docs/article-page/latest/all/glossary/raven-query-statistics). 
+Among the details you can get from the query statistics, you have: 
+
+* Whatever the index was stale or not.
+* The duration of the query on the server side.
+* The total number of results (regardless of paging).
+* The name of the index that this query run against.
+* The last document etag indexed by the index.
+* The timestamp of the last document indexed by the index.
+
+## Stale index?
+
+This is a very important fact: **RavenDB supports fully ACID writes but BASE reads**. 
```

Comment:
That isn't quite true, only queries are BASE, reads / writes by doc id are always ACID